_posts/2022-02-23-sparse-models.md

```diff
@@ -5,24 +5,31 @@ tags: []
 categories: [Machine Learning]
```

Comment:
I am reviewing this post as lying under the category 'Introductions to a new/old problem, tutorials, etc.' from [here](https://outline.skit.ai/doc/article-review-protocol-0cqBY8vW1U)

Reply:
First question that I am not very clear on is that this post is not adding something to what's already known. The 2-3 references' abstracts alone does the same job as this post. How do we defend this?

Reply:
If I understand correctly, the blogs are not supposed to be like "teaching material" and rather be closer to a research paper, as in present something new? I do not understand the goal of the blogs. If we only want to write about research outcomes, then it is possible after a paper is accepted.